spec/vcr_cassettes/Operation/Websocket_connection/should_bail_with_no_host.json

```diff
@@ -0,0 +1,3 @@
+[
+
+]
```

Comment:
Well, those cassettes were recorded via (WS)VCR, so I don't want to manually touch them.